ui/lib/core/addon/utils/duration-utils.ts

```diff
@@ -44,3 +46,15 @@ export const largestUnitFromSeconds = (seconds: number) => {
   }
   return unit;
 };
+
+// parses duration string ('3m') and returns seconds
+export const durationToSeconds = (duration: string) => {
+  // any number we assume is in seconds
+  if (typeof duration === 'number') return duration;
+  try {
+    return Duration.parse(duration).seconds();
+  } catch (e) {
+    // return false so parent can decide how to handle parsing error
+    return false;
```

Comment:
Parent should probably check `===` some value rather than falsey (`!returnedSeconds`) since the valid duration `0` would be falsey. So I would return null instead but that's just me

Reply:
I went back and forth between having null here and couldn't decide since they're both false. But that's a good point! Thank you